### DISCUSSION TITLE
Onboarding / Contributing documentation out of date

### DISCUSSION THREAD

**Discussion by ProLoser** [parcel-bundler/parcel#5793] [Q&A]
Please help me update it, I would like to try my fix for worklet.addModule() but I am unsure how to properly setup parcel for development. The instructions in the wiki and CONTRIBUTING.md are outdated, pointing to a third party website we no longer use. The document https://github.com/parcel-bundler/parcel/wiki/HEAD also failed to work as it seems it was written pre-monorepo?

I am now trying to figure out how to undo the instructions in that script ^

**Comment by mischnic** [comment]
> CONTRIBUTING.md are outdated, pointing to a third party website we no longer use

Where it that?

Also: what's would you like to see mentioned in there?

**Reply by ProLoser** [reply]
https://github.com/parcel-bundler/parcel/blob/master/CONTRIBUTING.md#spectrum

**Reply by ProLoser** [reply]
I would like to know how to setup parcel for local dev, the instructions under HEAD don't seem to work anymore.

**Reply by mischnic** [reply]
You should definitely look at CONTRIBUTING.md  in the v2 branch. I made some minor changes to that in https://github.com/parcel-bundler/parcel/pull/5794/files

I've removed/cleaned up the outdates content on the wiki (and changed the link to point to the v2 version of CONTRIBUTING)

**Reply by ProLoser** [reply]
Also I kinda prefer avoiding github wikis these days, as markdown in-repo tends to get updated more consistently

**Reply by ProLoser** [reply]
What I'm unclear is how to properly link the package. When I do it from the root it only links `@parcel/monorepo` do you have a script that auto-links the packages like `yarn workspaces run link`?

**Reply by mischnic** [reply]
These two might do what you want: https://github.com/parcel-bundler/parcel/blob/9b0f3ac77e70cfe31a304b6d4235b52f663cb9f8/package.json#L18-L19

Personally, I always use this hack: https://github.com/parcel-bundler/parcel/pull/5794/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R41

**Reply by ProLoser** [reply]
Is that all I have to do? and then link parcel in my project?

**Reply by ProLoser** [reply]
I'm seeing a lot of this:
```
No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.

gyp: No Xcode or CLT version detected!
```

**Reply by ProLoser** [reply]
I reinstalled xcode and my dependencies and it seems to be working
